### DISCUSSION TITLE
Release and Documentation

### DISCUSSION THREAD

**Discussion by edizaziz-dev** [paillave/Etl.Net#278] [General]
Hello! Firstly, thank you all for your arduous work and dedication. Secondly, judging by the readme file on Master, I understand the work for a release is complete and ready for release pending documentation. My question is, when? 

Some background, i'm working on a report for my employers who would like an off-the-shelf (bonus points for open source) ETL software that will take data from our warehouse (an MSSQL database) and split it out into various formats that a user can choose using a snazzy UI (built by us or off-the-shelf). 

I want to add this as part of my range of recommendations but i'm uncertain if the release will be ready "on time". ETL.NET certainly fits many of our use cases and I would like to know a rough timeline for release, if you can provide this?

Thanks in advanced and keep up the great work.

**Comment by paillave** [comment]
Hello, thanks for appreciating etlnet. I'm sorry, i didn't see there was any activity in the discussion. As you can see, the final version is released with a proper documentation. There is still lot of things to be done, but at the moment, I'm gonna focus on bugs and quick wins. You are more than welcome to bring your own amendments. Just let me know what are your projects.
